src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
```

Comment:
The class 'Figure' should not be abstract if it's only used to set behavior for classes. Consider using an interface if you need to define common behavior, and then have your concrete figure classes implement this interface.